e2e/utils/extended-playwright-test.ts

```
@@ -0,0 +1,35 @@
+ * Provides a temporary download directory for each test:
+ * 1. Creates a unique temp directory in the OS temp folder before the test runs
+ * 2. Provides the directory path to the test via the 'use' callback
+ * 3. Automatically cleans up (deletes) the directory after the test completes
+ *
+ * **IMPORTANT:** This must be exported as "test" (not renamed) to ensure ESLint's
+ * Playwright plugin rules work correctly with our custom fixtures.
+ */
+export const test = base.extend<{
+  downloadDir: string;
+}>({
+  downloadDir: async ({}, use) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)